src/content/components/notification/usage.md

```diff
@@ -27,6 +27,15 @@ We recommend that toast notifications automatically disappear after five seconds
 Icons may provide additional clarity. Icons should be placed to the left of a title. These glyphs (16x16) can be found in the [iconography](/guidelines/iconography/library) library.
 
 
+<image-component cols="8">
+
+![The three icon states](images/notification-usage-1.png)
+
+</image-component>
+
+## Placement
+
+
```

Comment:
It looks like there are two H2's next to each other here, is that intentional?

Reply:
There was a duplicate `## Placement` in the wrong spot. I deleted the extra.